FlyingSocks/Sources/Socket+Android.swift

```diff
@@ -99,6 +99,24 @@ extension Socket {
         return addr
     }
 
+    static func makeAbstractNamespaceUnix(name: String) -> Android.sockaddr_un {
+        var addr: sockaddr_un = Android.sockaddr_un()
+        addr.sun_family = sa_family_t(AF_UNIX)
+
+        _ = withUnsafeMutableBytes(of: &addr.sun_path) { raw in
+            raw.initializeMemory(as: CChar.self, repeating: 0)
+        }
+
+        let nameBytes = Array(name.utf8.prefix(107))
```

Comment:
I'm wondering if we should add a constant for this max as its actually used in a couple of places (macOS: 104, linux: 108)

```swift
extension Socket {
   static let sunPathMax = MemoryLayout.size(ofValue: sockaddr_un().sun_path)
}
```